local/scripts/dist.sh

```diff
@@ -33,6 +33,7 @@ git commit --allow-empty --message "$COMMIT_MESSAGE"
 if [ -n "$DIST_TAG" ]; then
 	echo "Tagging a release: $DIST_TAG"
 	git tag --force "$DIST_TAG"
+	git push --delete origin "$DIST_TAG"
```

Comment:
@kidunot89 AFAICT this will delete the current branch remotely, but will keep it locally along with the tag of the same name. Effectively leading to the same issue as before (?), where the `$DIST_BRANCH` refers both to a local tag and a local branch name.

Can you or @kasparsd confirm what the requirement is here?

The simplest solution would be name tags and branches differently, e.g. name the tag simply `3.5.1` and the branch `version-3.5.1`. Whether you want to keep it after a release or not is not really relevant. Having two names not only helps to disambiguate the ref types, but could also potentially help with debugging issues.

If we don't want to disambiguate the two, we could perhaps do this:

```
git push --force --set-upstream origin "refs/heads/${DIST_BRANCH}:refs/heads/{DIST_BRANCH}" --tags
```

Reply:
@dero I believe the goal is to rebuild the distribution tag with all needed assets for packagist.org

Reply:
This [error](https://travis-ci.com/github/xwp/stream/jobs/372078948#L474) was being caused by a collision with the remote tag when the rebuilt tag is pushed to the repo on the final line of the script.

Reply:
@dero See [this](https://github.com/xwp/stream/pull/1074)